2023-05-16 11:16:24,807 INFO    StreamThr :34356 [internal.py:wandb_internal():87] W&B internal server running at pid: 34356, started at: 2023-05-16 11:16:24.806731
2023-05-16 11:16:24,809 DEBUG   HandlerThread:34356 [handler.py:handle_request():144] handle_request: status
2023-05-16 11:16:24,819 INFO    WriterThread:34356 [datastore.py:open_for_write():85] open: C:\Users\shanthod\Downloads\ssl-relation-prediction-main\ssl-relation-prediction-main\src\wandb\run-20230516_111624-hqsscw0x\run-hqsscw0x.wandb
2023-05-16 11:16:24,821 DEBUG   SenderThread:34356 [sender.py:send():336] send: header
2023-05-16 11:16:24,826 DEBUG   SenderThread:34356 [sender.py:send():336] send: run
2023-05-16 11:16:25,612 INFO    SenderThread:34356 [dir_watcher.py:__init__():216] watching files in: C:\Users\shanthod\Downloads\ssl-relation-prediction-main\ssl-relation-prediction-main\src\wandb\run-20230516_111624-hqsscw0x\files
2023-05-16 11:16:25,612 INFO    SenderThread:34356 [sender.py:_start_run_threads():1067] run started: hqsscw0x with start time 1684206984.808376
2023-05-16 11:16:25,612 DEBUG   SenderThread:34356 [sender.py:send_request():363] send_request: summary_record
2023-05-16 11:16:25,614 DEBUG   HandlerThread:34356 [handler.py:handle_request():144] handle_request: check_version
2023-05-16 11:16:25,615 INFO    SenderThread:34356 [sender.py:_save_file():1321] saving file wandb-summary.json with policy end
2023-05-16 11:16:25,615 DEBUG   SenderThread:34356 [sender.py:send_request():363] send_request: check_version
2023-05-16 11:16:25,675 DEBUG   HandlerThread:34356 [handler.py:handle_request():144] handle_request: run_start
2023-05-16 11:16:25,700 DEBUG   HandlerThread:34356 [system_info.py:__init__():31] System info init
2023-05-16 11:16:25,700 DEBUG   HandlerThread:34356 [system_info.py:__init__():46] System info init done
2023-05-16 11:16:25,700 INFO    HandlerThread:34356 [system_monitor.py:start():151] Starting system monitor
2023-05-16 11:16:25,701 INFO    SystemMonitor:34356 [system_monitor.py:_start():116] Starting system asset monitoring threads
2023-05-16 11:16:25,702 INFO    HandlerThread:34356 [system_monitor.py:probe():172] Collecting system info
2023-05-16 11:16:25,702 DEBUG   HandlerThread:34356 [system_info.py:probe():195] Probing system
2023-05-16 11:16:25,702 INFO    SystemMonitor:34356 [interfaces.py:start():168] Started cpu
2023-05-16 11:16:25,713 INFO    SystemMonitor:34356 [interfaces.py:start():168] Started disk
2023-05-16 11:16:25,714 INFO    SystemMonitor:34356 [interfaces.py:start():168] Started memory
2023-05-16 11:16:25,725 INFO    SystemMonitor:34356 [interfaces.py:start():168] Started network
2023-05-16 11:16:25,733 DEBUG   HandlerThread:34356 [git.py:repo():40] git repository is invalid
2023-05-16 11:16:25,733 DEBUG   HandlerThread:34356 [system_info.py:probe():241] Probing system done
2023-05-16 11:16:25,733 DEBUG   HandlerThread:34356 [system_monitor.py:probe():181] {'os': 'Windows-10-10.0.19041-SP0', 'python': '3.9.0', 'heartbeatAt': '2023-05-16T03:16:25.712418', 'startedAt': '2023-05-16T03:16:24.787473', 'docker': None, 'cuda': None, 'args': None, 'state': 'running', 'program': 'C:\\Users\\shanthod\\Downloads\\ssl-relation-prediction-main\\ssl-relation-prediction-main\\src\\main.py', 'codePath': 'main.py', 'host': 'shanthod0QG3L', 'username': 'shanthod', 'executable': 'C:\\Users\\shanthod\\AppData\\Local\\Programs\\Python\\Python39\\python.exe', 'cpu_count': 4, 'cpu_count_logical': 8, 'cpu_freq': {'current': 1805.0, 'min': 0.0, 'max': 1805.0}, 'cpu_freq_per_core': [{'current': 1805.0, 'min': 0.0, 'max': 1805.0}], 'disk': {'total': 474.31444931030273, 'used': 411.0939140319824}, 'memory': {'total': 31.72928237915039}}
2023-05-16 11:16:25,733 INFO    HandlerThread:34356 [system_monitor.py:probe():182] Finished collecting system info
2023-05-16 11:16:25,733 INFO    HandlerThread:34356 [system_monitor.py:probe():185] Publishing system info
2023-05-16 11:16:25,733 DEBUG   HandlerThread:34356 [system_info.py:_save_pip():51] Saving list of pip packages installed into the current environment
2023-05-16 11:16:25,740 DEBUG   HandlerThread:34356 [system_info.py:_save_pip():67] Saving pip packages done
2023-05-16 11:16:25,745 INFO    HandlerThread:34356 [system_monitor.py:probe():187] Finished publishing system info
2023-05-16 11:16:25,781 DEBUG   SenderThread:34356 [sender.py:send():336] send: files
2023-05-16 11:16:25,781 INFO    SenderThread:34356 [sender.py:_save_file():1321] saving file wandb-metadata.json with policy now
2023-05-16 11:16:25,801 DEBUG   HandlerThread:34356 [handler.py:handle_request():144] handle_request: stop_status
2023-05-16 11:16:25,802 DEBUG   SenderThread:34356 [sender.py:send_request():363] send_request: stop_status
2023-05-16 11:16:26,127 DEBUG   SenderThread:34356 [sender.py:send():336] send: telemetry
2023-05-16 11:16:26,135 DEBUG   SenderThread:34356 [sender.py:send():336] send: config
2023-05-16 11:16:26,135 DEBUG   SenderThread:34356 [sender.py:send():336] send: telemetry
2023-05-16 11:16:26,135 DEBUG   SenderThread:34356 [sender.py:send_request():363] send_request: summary_record
2023-05-16 11:16:26,140 INFO    SenderThread:34356 [sender.py:_save_file():1321] saving file wandb-summary.json with policy end
2023-05-16 11:16:26,141 DEBUG   SenderThread:34356 [sender.py:send_request():363] send_request: summary_record
2023-05-16 11:16:26,147 INFO    SenderThread:34356 [sender.py:_save_file():1321] saving file wandb-summary.json with policy end
2023-05-16 11:16:26,627 INFO    Thread-17 :34356 [dir_watcher.py:_on_file_created():275] file/dir created: C:\Users\shanthod\Downloads\ssl-relation-prediction-main\ssl-relation-prediction-main\src\wandb\run-20230516_111624-hqsscw0x\files\wandb-summary.json
2023-05-16 11:16:26,628 INFO    Thread-17 :34356 [dir_watcher.py:_on_file_created():275] file/dir created: C:\Users\shanthod\Downloads\ssl-relation-prediction-main\ssl-relation-prediction-main\src\wandb\run-20230516_111624-hqsscw0x\files\wandb-metadata.json
2023-05-16 11:16:26,628 INFO    Thread-17 :34356 [dir_watcher.py:_on_file_created():275] file/dir created: C:\Users\shanthod\Downloads\ssl-relation-prediction-main\ssl-relation-prediction-main\src\wandb\run-20230516_111624-hqsscw0x\files\output.log
2023-05-16 11:16:26,629 INFO    Thread-17 :34356 [dir_watcher.py:_on_file_created():275] file/dir created: C:\Users\shanthod\Downloads\ssl-relation-prediction-main\ssl-relation-prediction-main\src\wandb\run-20230516_111624-hqsscw0x\files\requirements.txt
2023-05-16 11:16:27,373 INFO    wandb-upload_0:34356 [upload_job.py:push():138] Uploaded file C:\Users\shanthod\AppData\Local\Temp\tmp3r7b9y1ewandb\e5yduspw-wandb-metadata.json
2023-05-16 11:16:28,629 INFO    Thread-17 :34356 [dir_watcher.py:_on_file_modified():292] file/dir modified: C:\Users\shanthod\Downloads\ssl-relation-prediction-main\ssl-relation-prediction-main\src\wandb\run-20230516_111624-hqsscw0x\files\output.log
2023-05-16 11:16:30,172 DEBUG   HandlerThread:34356 [handler.py:handle_request():144] handle_request: status_report
2023-05-16 11:16:35,371 DEBUG   HandlerThread:34356 [handler.py:handle_request():144] handle_request: status_report
2023-05-16 11:16:36,716 INFO    Thread-17 :34356 [dir_watcher.py:_on_file_modified():292] file/dir modified: C:\Users\shanthod\Downloads\ssl-relation-prediction-main\ssl-relation-prediction-main\src\wandb\run-20230516_111624-hqsscw0x\files\output.log
2023-05-16 11:16:40,423 DEBUG   HandlerThread:34356 [handler.py:handle_request():144] handle_request: status_report
2023-05-16 11:16:40,871 DEBUG   HandlerThread:34356 [handler.py:handle_request():144] handle_request: keepalive
2023-05-16 11:16:40,874 DEBUG   HandlerThread:34356 [handler.py:handle_request():144] handle_request: keepalive
2023-05-16 11:16:43,835 ERROR   memory    :34356 [interfaces.py:monitor():129] Failed to sample metric: process no longer exists (pid=9780)
2023-05-16 11:16:43,835 ERROR   memory    :34356 [interfaces.py:monitor():129] Failed to sample metric: process no longer exists (pid=9780)
2023-05-16 11:16:43,882 ERROR   cpu       :34356 [interfaces.py:monitor():129] Failed to sample metric: process no longer exists (pid=9780)
2023-05-16 11:16:43,889 ERROR   cpu       :34356 [interfaces.py:monitor():129] Failed to sample metric: process no longer exists (pid=9780)
2023-05-16 11:16:45,478 DEBUG   HandlerThread:34356 [handler.py:handle_request():144] handle_request: status_report
2023-05-16 11:16:45,836 ERROR   memory    :34356 [interfaces.py:monitor():129] Failed to sample metric: process no longer exists (pid=9780)
2023-05-16 11:16:45,836 ERROR   memory    :34356 [interfaces.py:monitor():129] Failed to sample metric: process no longer exists (pid=9780)
2023-05-16 11:16:45,852 INFO    MainThread:34356 [internal.py:handle_exit():77] Internal process exited
